### ISSUE TITLE
Compiling wasm-bindgen-futures v0.4.40 gives `error[E0658]: use of unstable library feature 'stdarch_wasm_atomic_wait'`

### ISSUE THREAD

**Issue by fzyzcjy** [rustwasm/wasm-bindgen#3830]
### Describe the Bug

Hi thanks for the toolkit! However, it seems that I cannot compile it (in https://github.com/fzyzcjy/flutter_rust_bridge) and it gives the error below.
Related PR: https://github.com/fzyzcjy/flutter_rust_bridge/pull/1727

It seems that https://github.com/rustwasm/wasm-bindgen/pull/3827 fixes this issue, but https://crates.io/crates/wasm-bindgen-futures/versions's last release is 25 days ago, so maybe it needs to release a new version.

### Steps to Reproduce

Compile it:

```
   Compiling wasm-bindgen-futures v0.4.40
error[E0658]: use of unstable library feature 'stdarch_wasm_atomic_wait'
  --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wasm-bindgen-futures-0.4.40/src/task/multithread.rs:38:13
   |
38 |             core::arch::wasm32::memory_atomic_notify(
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #77839 <https://github.com/rust-lang/rust/issues/77839> for more information
   = help: add `#![feature(stdarch_wasm_atomic_wait)]` to the crate attributes to enable
   = note: this compiler was built on 2024-02-05; consider upgrading it if it is out of date

error[E0635]: unknown feature `stdsimd`
  --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wasm-bindgen-futures-0.4.40/src/lib.rs:33:49
   |
33 | #![cfg_attr(target_feature = "atomics", feature(stdsimd))]
   |                                                 ^^^^^^^

Some errors have detailed explanations: E0635, E0658.
For more information about an error, try `rustc --explain E0635`.
error: could not compile `wasm-bindgen-futures` (lib) due to 2 previous errors
```

### Actual Behavior

A clear and concise description of what actually happened.

**Comment by daxpedda** [comment]
Indeed #3827 fixes the issue and we have already released a new version in #3829 but having some Crates.io deployment issue.
So some patience will be required.

**Comment by fzyzcjy** [comment]
I see, thank you! (I saw that "0.2.91" issue, but wrongly thought it was unrelated to futures, since it is 0.4.x)
